server/container/guice/configuration/src/main/java/org/apache/james/utils/PropertiesProvider.java

```diff
@@ -43,11 +43,20 @@
 import com.google.common.base.Strings;
 
 public class PropertiesProvider {
-
     private static final Logger LOGGER = LoggerFactory.getLogger("org.apache.james.CONFIGURATION");
     private static final char COMMA = ',';
     private static final String COMMA_STRING = ",";
 
+    public static Configuration getConfiguration(File propertiesFile) throws ConfigurationException {
```

Comment:
:thinking: move from private instance method to public static ... ? that's a huge scoping increase ... 

I'm not saying it's wrong but it feels really weird and it seems to be done exclusively for tests, isn't that a bit extreme ?

Reply:
Maybe I could annotate it with `@VisibleForTesting` to make the intent clear?

Reply:
> I'm not saying it's wrong but it feels really weird and it seems to be done exclusively for tests, isn't that a bit extreme ?

I'm trying to address a diverging beaviour between tests and 'production', initializing all the wiring to get a propertyProvider felt overwhelming...

Reply:
I found a better way...